src/Momento.Sdk/Config/Middleware/IMiddleware.cs

```diff
@@ -36,6 +38,11 @@ Func<Metadata> GetTrailers
 /// </summary>
 public interface IMiddleware
 {
+    /// <summary>
+    /// Add headers to gRPC stream requests since middlewares are incompatible with gRPC streams.
+    /// </summary>
+    public IList<Tuple<string, string>> AddStreamRequestHeaders();
+
```

Comment:
Why do we need this? All of the implementations are trivial, ie return an empty list.